apps/studio/components/ui/AdvisorPanel/AdvisorDetail.tsx

```diff
@@ -0,0 +1,40 @@
+import LintDetail from 'components/interfaces/Linter/LintDetail'
+import { Lint } from 'data/lint/lint-query'
+import { Notification } from 'data/notifications/notifications-v2-query'
+import { NotificationDetail } from './NotificationDetail'
+import { AdvisorItem } from './AdvisorPanelHeader'
+
+interface AdvisorDetailProps {
+  item: AdvisorItem
+  projectRef: string
+  onUpdateNotificationStatus?: (id: string, status: 'archived' | 'seen') => void
+}
+
+export const AdvisorDetail = ({
+  item,
+  projectRef,
+  onUpdateNotificationStatus = () => {},
```

Comment:
Small nit: use `noop()` (you'll need to add an import) instead of `() => {}` in React component default arguments. If you ever pass `onUpdateNotificationStatus` to a `useEffect()` dependency array with the default value, it'll cause the `useEffect()` to rerun on every render. A fun footgun of React 😅

```suggestion
  onUpdateNotificationStatus = noop,
```